tests/integration_tests.rs

```diff
@@ -1153,6 +1153,52 @@ fn show_all_mode() {
         .stderr("");
 }
 
+#[test]
+fn show_all_tabstops() {
+    bat()
+        .arg("tabs.txt")
+        .arg("--show-all")
+        .arg("--tabs=4")
+        .arg("--style=plain")
+        .assert()
+        .success()
+        .stdout(
+            "├──┤1├─┤2├─┤3├─┤4␊
+1├─┤?␊
+22├┤?␊
+333↹?␊
+4444├──┤?␊
+55555├─┤?␊
+666666├┤?␊
+7777777↹?␊
+88888888├──┤?␊
+",
+        );
+}
+
+#[test]
+fn show_all_tabstops_long() {
```

Comment:
Feel free to choose very long test names (I have to admit that some other test names could be improved as well). Maybe:

`show_all_extends_tab_markers_to_next_tabstop`